docs/node-guide/quick-start.md

```diff
@@ -23,28 +36,28 @@ This command could take some time since it will download a very large LLM file.
 gaianet init
 ```
 
-Finally, start your node.
+4. Use the following command to start your node:
 
 ```
 gaianet start
 ```
 
-Upon a successful start, the command prints a public URL for the node. You can open a browser to that URL to see the node information and then chat with the AI agent on the node.
+5. A successful start prints a public URL for the node. Opening a browser to that URL will display the node information and allow you to chat with the AI agent on the node. 
```

Comment:
I think it shouldn't be concluded as a list. After the `gaianet start`, the process of starting a node is done.